packages/components/psammead-inline-link/src/index.stories.jsx

```diff
@@ -24,4 +55,98 @@ storiesOf('Components|InlineLink', module)
       </>
     ),
     { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'inline-link inside a regular paragraph',
+    ({ text, longText, script, service }) => (
+      <>
+        <RegularParagraph script={script} service={service}>
```

Comment:
If we extract this into a separate function, we can re-use it in line `115` simply by passing in `style` as a prop.

I'm thinking something like 
```
const renderRegularParagraph = ({script, service,  longText, text}) =>(
      <RegularParagraph script={script} service={service}>
          {longText}{' '}
          <InlineLink href="https://www.bbc.com/news">{text}</InlineLink>{' '}
          {longText}
        </RegularParagraph>
      </>
)
```